hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestCDCForSparkSQL.scala

```diff
@@ -56,7 +56,7 @@ class TestCDCForSparkSQL extends HoodieSparkSqlTestBase {
       Seq(OP_KEY_ONLY, DATA_BEFORE, DATA_BEFORE_AFTER).foreach { loggingMode =>
         withTempDir { tmp =>
           val tableName = generateTableName
-          val basePath = s"${tmp.getCanonicalPath}/$tableName"
+          val basePath = s"${tmp.getCanonicalPath}/$tableName".replaceAll("\\\\", "\\/")
```

Comment:
wouldn't this be a general problem for all test cases? maybe can you make a sweep and fix all occurrence of such issue?

Reply:
> wouldn't this be a general problem for all test cases? maybe can you make a sweep and fix all occurrence of such issue?

@xushiyan i had changed other testcase for windows system env，have a review please